src/integration-test/java/org/openlmis/requisition/web/RequisitionControllerIntegrationTest.java

```diff
@@ -1163,11 +1168,59 @@ public void shouldGetApprovedRequisitionsWithSortByFilterByAndPaging() {
     // when
     PageImplRepresentation response = restAssured.given()
         .header(HttpHeaders.AUTHORIZATION, getTokenHeader())
-        .queryParam("filterValue", filterValue)
-        .queryParam("filterBy", filterBy)
-        .queryParam("sort", sortBy)
-        .queryParam("page", page)
-        .queryParam("size", size)
+        .queryParam(FILTER_VALUE, filterValue)
+        .queryParam(FILTER_BY, filterBy)
+        .queryParam(SORT, sortBy)
+        .queryParam(PAGE, page)
+        .queryParam(SIZE, size)
+        .contentType(MediaType.APPLICATION_JSON_VALUE)
+        .when()
+        .get(APPROVED_REQUISITIONS_SEARCH_URL)
+        .then()
+        .statusCode(200)
+        .extract().as(PageImplRepresentation.class);
+
+    // then
+    assertEquals(1, response.getContent().size());
+    assertThat(RAML_ASSERT_MESSAGE, restAssured.getLastReport(), RamlMatchers.hasNoViolations());
+  }
+
+  @Test
+  public void shouldGetApprovedRequisitionsWithSortByMultipleFilterValues() {
```

Comment:
Can you remove code duplication from this test and above?

Reply:
Which parts do you think would make sense extracting as private methods, such that they do not interrupt usage of variables in the test? @palbecki

Reply:
Can we extract everything and pass to method param filterValue? If not then just extract lines that are duplicated for few private methods

Reply:
I extracted right mocking and reused facility mocking. Anything else?

Reply:
looks good, thanks